rpm/kiwi.spec

```diff
@@ -22,6 +22,13 @@
 %define perl_version    %(eval "`%{__perl} -V:version`"; echo $version)
 %endif
 
+# in TW and SLE12SP2 python-kiwi provides the kiwi-tools package
+%if 0%{?suse_version} > 1320 || 0%{?sle_version} >= 120200
```

Comment:
I'm not the expert but does this check covers all places where python-kiwi exists ? I'm sure the sle12 match is complete but I can't tell about Leap